change/@azure-msal-browser-936e7906-4429-4deb-a6b5-6cee0ee45120.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I don't think these change files should be included. They were deleted before the last v2 release went out as they were integrated into changelogs.

Reply:
I realize this was the state of dev before the release, we may need to pull from dev again into msal-browser-v3 after merging this PR.

Reply:
They will be deleted automatically during next rebase.